spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/StreamService.java

```diff
@@ -30,6 +30,20 @@
  * @author Christian Tzolov
  */
 public interface StreamService {
+
+	/**
+	 * Create a new stream.
+	 *
+	 * @param streamName stream name
+	 * @param dsl DSL definition for stream
+	 * @param deploy if {@code true}, the stream is deployed upon creation (default is
+	 * {@code false})
+	 * @return the created stream definition already exists
+	 * @throws InvalidStreamDefinitionException if there errors in parsing the stream DSL,
+	 * resolving the name, or type of applications in the stream
+	 */
+	StreamDefinition createStream(String streamName, String dsl, boolean deploy);
```

Comment:
Should we add `throws InvalidStreamDefinitionException` for this method definition?

Reply:
Given that `InvalidStreamDefinitionException` is a Runtime exception i believe we should not add it explicitly?

Reply:
I agree and having it in the Javadoc is sufficient. We can just fix the import part of this exception in this class file.